### ISSUE TITLE
CDN css file missing selectors

### ISSUE THREAD

**Issue by glossyweb** [tomickigrzegorz/show-more#63]
CDN css file dont have a 
.hidden {
  display: none;
}

so show hide not work )

**Comment by tomickigrzegorz** [comment]
Yes, that's right, it's on purpose. Usually you use some css library that has a class display: none; e.g. bootstrap d-none. Why duplicate styles ;)
